IaC/PnP/monitoring/aks/online/aks_secure_baseline/ngsa-memory-with-crd/ngsa-memory.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: mock
+  labels:
+    name: mock
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ngsa-memory
+  namespace: mock
+  labels:
+    app.kubernetes.io/name: ngsa-memory
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ngsa-memory
+  template:
+    metadata:
+      labels:
+        app: ngsa-memory
+        version: beta-0329-1641
+        deploy: 21-03-29-16-41-59
```

Comment:
[minor]
The container is using the `beta` label which changes over time. Are these labels here needed since they'll be out of date when the next release happens?

Reply:
Not sure what the labels are used for. Since these are just labels, we can just remove them for the spike.
@AAkindele Thoughts?

Reply:
At first it was in the spikes folder, but this is tied to the monitoring deployment. It doesn't deploy the traefik per se, rather it creates custom ingressroute+middleware for traefik to use.

If we were to move the folder what should be the folder name? "traefik-ingressroute-middleware"? "traefik-with-crd"?